.github/workflows/pull-request-check.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [10, 12, 14, 15]
+        node-version: [12, 14]
```

Comment:
We also want to add 16. It is already in the "current state" and we better test for it. https://nodejs.org/en/about/releases/